ktor-server/ktor-server-core/jvm/src/io/ktor/server/response/ApplicationResponseFunctionsJvm.kt

```diff
@@ -41,14 +42,26 @@ public suspend fun ApplicationCall.respondOutputStream(
 }
 
 /**
- * Responds to a client with a contents of a file with the name [fileName] in the [baseDir] folder
+ * Responds to a client with a contents of a file with the name [relativePath] in the [baseDir] folder
  */
 public suspend fun ApplicationCall.respondFile(
     baseDir: File,
-    fileName: String,
+    relativePath: String,
```

Comment:
Couldn't this change cause problems with backward compatibility? In cases when named arguments are used.

Reply:
I thought about it, and it seems okay if it goes to the 3.0.0 release, but maybe it's an unneeded change so will revert